bot/cogs/reddit.py

```diff
@@ -290,4 +290,7 @@ def channel(self) -> TextChannel:
 
 def setup(bot: Bot) -> None:
     """Load the Reddit cog."""
-    bot.add_cog(Reddit(bot))
+    if None not in (RedditConfig.client_id, RedditConfig.secret):
+        bot.add_cog(Reddit(bot))
+        return
+    log.error("Credentials not provided, cog not loaded.")
```

Comment:
Just a matter of preference, but I think these would read nicer as:
```python
if None in (...):
  log.error(...
  return
bot.add_cog(...
```
Feel free to keep as is though.

Reply:
I stylized it with the intended case as the first one (although the eyes fall more onto the log...), maybe a third opinion?

Reply:
I prefer @kwzrd's